addons/website/views/snippets/snippets.xml

```diff
@@ -1001,20 +1001,19 @@
         </we-button-group>
     </div>
 
-    <!-- Min height of section -->
-    <div data-option-name="minHeight" data-selector="section">
```

Comment:
You removed that data-option-name without adapting the code related to it: I grepped it, there are some occurences in themes, so you need to adapt that in PR with the same name in those theme repo (or found another solution here?). To check.

Reply:
I checked but couldn't find any occurrence of `data-option-name` or `minHeight` in `design-themes` repo.

Reply:
How did you check? :roll_eyes: 

![Screenshot from 2021-06-14 11-47-51](https://user-images.githubusercontent.com/10338094/121873222-7157a800-cd06-11eb-8544-2d837fb0d1c9.png)